appjs/templates/project_list.ejs

```diff
@@ -62,6 +48,16 @@
             <% }) %>
             </select>
           </div>
+            <label class="sr-only" for="search">Search query</label>
+            <div class="btn-group">
+              <input type="text" class="form-control"
+                     name="search" id="search" placeholder="Search"
+                     <% if(query.search) { %>value="<%=query.search %>"<% } %>>
+            <% if(query.search) { %>
```

Comment:
Doesn't the clear link only show up if you've done a search? How do you clear if you've applied a theme filter?